gammapy/spectrum/tests/test_spectrum_fit.py

```diff
@@ -10,6 +10,10 @@
 from ...utils.testing import requires_dependency, requires_data, SHERPA_LT_4_8
 from astropy.utils.compat import NUMPY_LT_1_9
 
+
+#TODO : PHACount should not expect a BIN_LO and BIN_HI columns
+#https://travis-ci.org/gammapy/gammapy/jobs/136170992#L939 
```

Comment:
Tests on travis-ci passed. @registerrier wants to start new work that partly builds on this pull request.
So the easiest would be to merge this PR now, if that's OK with you @joleroi ?
The main concern I have is that this PR adds an `xfail` to `test_spectral_fit` which I know you've been working on.
Maybe this has to wait a bit and then the xfail has to be removed here before it should go in?